src/ripple/app/tx/impl/Transactor.h

```diff
@@ -169,6 +168,11 @@ class Transactor
 
     virtual TER doApply () = 0;
 
+	static
+	XRPAmount
+	calculateFee(Application& app, std::uint64_t const baseFee,
+		Fees const& fees, ApplyFlags flags);
+
```

Comment:
Formatting is off here.

If this function is going to use const value parameters, it you should do it for all the const value parameters in the function signature. I.e. The `flags` parameter should be `ApplyFlags const flags`. I think const value parameters like `std::uint64_t const` are weird and most C++ code doesn't do that. However, it's hard to call it "wrong" or make a strong argument against it. So while I vote "no const value params" it's OK with me if you want to keep it.

Reply:
I like `const` and I'm coming to the conclusion that I tend to overuse it.